Readme.md

```diff
@@ -226,6 +226,23 @@ You can run any kind of code in parallel with -e / --exec
 <tr><td>Rails-ActionPack</td><td>88s</td><td>53s</td><td>44s</td></tr>
 </table>
 
+`TEST_ENV_NUMBER`
+=================
+
```

Comment:
I made it briefer but kept a little more detail. Let me know if you want further changes.